content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/inventory_valuation_config.rst

```diff
@@ -117,26 +134,52 @@ value, and any product moves from then on affect the average cost, and the cost
 change. If the value in the :guilabel:`Cost` field on a product form is changed manually, Odoo will
 generate a corresponding record in the *Inventory Valuation* report.
 
-.. _inventory/management/config-inventory-valuation:
+.. note::
+   It is possible to use different valuation settings for different product categories.
 
-Inventory valuation
--------------------
+.. _inventory/warehouses_storage/accounting-types:
 
-Inventory valuation in Odoo can be set to be updated manually or automatically. While *Expense*
-accounts apply to both, the *Stock Input* and *Stock Output* accounts are only used for automated
-valuation.
+Types of accounting
+===================
 
-Refer to the :ref:`Expense <inventory/management/expense-account>` and :ref:`Stock input/output
-<inventory/management/stock-account>` sections for details on configuring each account type.
+With automated inventory valuation set up, the generated journal entries depend on the chosen
+accounting mode: *Continental* or *Anglo-Saxon*.
 
-.. _inventory/management/expense-account:
+.. tip::
+   Verify the accounting mode by activating the :ref:`developer-mode` and navigating to
+   :menuselection:`Accounting app --> Configuration --> Settings`.
+
+   Then, in the search bar, look for :guilabel:`Anglo-Saxon Accounting`, to see if the feature is
+   enabled. If it is **not** enabled, Continental accounting mode is in use.
+
+   .. image:: inventory_valuation_config/anglo-saxon.png
+      :align: center
+      :alt: Show the Anglo-Saxon accounting mode feature.
+
+In *Anglo-Saxon* accounting, the costs of goods sold (COGS) are reported when products are sold or
+delivered. This means that the cost of a good is only recorded as an expense when a customer is
+invoiced for a product. So for **manual** valuation method, set the :guilabel:`Expense Account` to
+`Stock Valuation` for the current asset type; for **automatic** valuation method, set the
+:guilabel:`Expense Account` to an *Expenses* or a *Cost of Revenue* type (e.g. `Cost of Production`,
+`Cost of Goods Sold`, etc.).
```

Comment:
```suggestion
delivered. This means the cost of a good is only recorded as an expense when a customer is
invoiced for a product.

So, for **manual** valuation method, set the :guilabel:`Expense Account` to *Stock Valuation* for
the current asset type; for **automatic** valuation method, set the :guilabel:`Expense Account` to
an *Expenses* or a *Cost of Revenue* type (e.g. *Cost of Production*, *Cost of Goods Sold*, etc.).
```
suggesting the use of italics here because they are proper setting options that can be chosen BUT aren't being shown directly on the page at this point. However, if you disagree, or if they are actually options that are manually typed in by the user, feel free to keep it as is